services.yaml

```diff
@@ -909,49 +926,197 @@ paths:
           description: NOT FOUND
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+    get:
+      tags:
+        - View Folders
+        - VFs
+      summary: "List View Folders"
+      description: "List View Folders"
+      responses:
+        200:
+          description: List of VF names
+          schema:
+            properties:
+              vf_names:
+                type: "array"
+                items:
+                  type: "object"
+                  properties:
+                    name:
+                      $ref: "#/definitions/Folder"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders/{folder_id}/views:
```

Comment:
where is the `/services/{service_id}/folders/{folder_id}` path? Even if it's not implemented we should at least put it as "TBD" otherwise it doesn't feel complete

Reply:
added path with get and delete calls (marked as 'not implemented yet')